packages/screens/AdministrationDashboard/AdministrationDashboardScreen.tsx

```diff
@@ -0,0 +1,568 @@
+import moment from "moment";
```

Comment:
I think this one is draft and it contains Dummy data for testing. Just remind that in general, we are trying to keep the file as short as possible, if it's too long we should think about to split in to several components

Reply:
Fixed, PR updated